Userland/Libraries/LibCore/Object.h

```diff
@@ -205,9 +205,11 @@ class Object
     virtual void did_end_inspection() { }
 
 private:
+    using TimerId = int;
```

Comment:
If both Core::Object and Core::EventLoop have the same concept for TimerID, why not use the same exact type for each instance? i.e. Core::EventLoop should be using a DistinctNumeric defined with the macro I mentioned above in Object.h. e.g. Core::TimerId.